client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -36,7 +36,7 @@ const StatsNavigation = ( props ) => {
 					initialSelected="site"
 					options={ [
 						{ value: 'site', label: translate( 'Site' ) },
-						{ value: 'store', label: translate( 'Store' ), path: `/store/stats/${ slug }` },
+						{ value: 'store', label: translate( 'Store' ), path: `/store/stats/orders/7day/sales_by_product/${ slug }` },
```

Comment:
Huh! I was playing around with this and wondering what was going on with the previous commit. This looks better :)